### PR TITLE
Add docs about configuring PETSc

### PR DESCRIPTION
**Pull request by jedbrown**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,8 +35,17 @@ To configure, build, and install RDycore:
    ```
    git submodule update --init --recursive
    ```
-2. Make sure you set your `PETSC_DIR` and `PETSC_ARCH` environment variables
-   to refer to your existing installation of PETSc.
+2. Make sure you set your `PETSC_DIR` and `PETSC_ARCH` environment variables to
+   refer to your existing installation of PETSc.
+   * PETSc must be configured with `--download-muparser`.
+   * If you intend to use [libCEED](https://libceed.org) (higher performance and
+     allows targeting GPUs), please configure PETSc with `--download-libceed`.
```
Comment (bishtgautam) [inline README.md:42]: I believe libCEED is required and not optional for RDycore.

**Comment by jeff-cohere** [comment]
We have a pretty comprehensive treatment of this process [in our static docs site](https://rdycore.github.io/RDycore/common/installation.html). Should we link to that here?

**Comment by jedbrown** [comment]
I think the question is which site will be maintained when requirements change. I didn't think of the static site when a student was running into issues building. Shall we delete the build instructions in the repo and just link, or shall we move build-from-source docs from the static site to the repo?

**Comment by bishtgautam** [comment]
Okay, I see @jedbrown's point. I think we should modify the `README.md` to point to the installation instructions listed in `docs/common/installation.md` (in the repo) and let the reader know those same instructions can be viewed online on https://rdycore.github.io/RDycore/common/installation.html.

**Comment by jeff-cohere** [comment]
I completely agree.

**Pull request closed without merging**